### PR TITLE
Task #58551 [LiconKA] Function-8 : Upload Audio to User Google Drive

### PR DESCRIPTION
**Pull request by liconrepo**

### CODE REVIEW

File: app/src/main/java/com/training/soundstudio/util/UIUtil.java [hunk 1]
```diff
@@ -65,4 +66,25 @@ public static void showDialogNotify(final Activity activity, final String title,
...
+                activity.getString(R.string.network_fix),
+                getDefaultDismissListener(),
+                getFixNetworkDismissListener(activity));
+    }
+
+    private static DialogInterface.OnClickListener getFixNetworkDismissListener(final Activity activity) {
+        DialogInterface.OnClickListener onCancelClickListener = new DialogInterface.OnClickListener() {
+            @Override
+            public void onClick(DialogInterface dialog, int id) {
+                activity.startActivityForResult(new Intent(android.provider.Settings.ACTION_SETTINGS), 0);
```
Comment (b1acKr0se) [inline app/src/main/java/com/training/soundstudio/util/UIUtil.java]: Please create a constant for the requestCode

File: app/src/main/java/com/training/soundstudio/BaseDriveActivity.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        Log.e(TAG, String.format(getString(R.string.drive_google_api_client_connection_failed),
+                result.toString()));
+        if (!result.hasResolution()) {
+            GoogleApiAvailability.getInstance().getErrorDialog(this, result.getErrorCode(), 0).show();
+            return;
+        }
+        try {
+            result.startResolutionForResult(this, REQUEST_CODE_RESOLUTION);
+        } catch (SendIntentException exception) {
+            Log.e(TAG, String.format(getString(R.string.drive_google_api_client_connection_exception),
```
Comment (b1acKr0se) [inline app/src/main/java/com/training/soundstudio/BaseDriveActivity.java]: please remove all logs in this file

**Comment by b1acKr0se** [comment]
OK

**Comment by nvmanh** [comment]
OK!

**Pull request merged**